test/smoketest/smoketest-update.bash

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -eux -o pipefail
+
+# Put these somewhere else before running via travis
+export TRAVIS_BUILD_DIR=`git rev-parse --show-toplevel`
```

Comment:
Putting the TEST directory in travis.yml just seems like a good hint to folks reviewing the travis setup that there is stuff close to being useful there. If you like I'll take it out, but it seems useful to me as is.